std/digest/digest.d

```diff
@@ -840,7 +840,7 @@ ref T[N] asArray(size_t N, T)(ref T[] source, string errorMsg = "")
      return *cast(T[N]*) source.ptr;
 }
 
-/**
+/*
  * This helper is used internally in the WrapperDigest template, but it might be
  * useful for other purposes as well. It returns the length (in bytes) of the hash value
  * produced by T.
```

Comment:
That change might be controversial, but there weren't any tests for `digestLength` and the documentation starts with "this helper is used internally", so this change wouldn't expose it on dlang.org.
What's your opinion on this @jpf91? Should we better reword the documentation and add tests?